deployment/src/strongmind_deployment/container.py

```diff
@@ -520,26 +519,20 @@ def dns(self, name, stack):
         full_name = f"{name}.{domain}"
         zone_id = self.kwargs.get('zone_id', 'b4b7fec0d0aacbd55c5a259d1e64fff5')
         lb_dns_name = self.kwargs.get('load_balancer_dns_name',
-                                      self.load_balancer.load_balancer.dns_name)  # pragma: no cover
+                                      self.load_balancer.dns_name)  # pragma: no cover
         if self.kwargs.get('cname', True):
             self.cname_record = Record(
                 'cname_record',
                 name=name,
                 type='CNAME',
+                allow_overwrite=True,
```

Comment:
Did we check that this worked? If so, great!

Reply:
Yes this fixed the issue that caused me to take down repo dashboard.